data/model/user.py

```diff
@@ -1271,6 +1272,10 @@ def delete_user(user, queues):
     # Delete non-repository related items.
     _delete_user_linked_data(user)
 
+    # Delete the namespace size if it exists
+    with db_transaction():
+        QuotaNamespaceSize.delete().where(QuotaNamespaceSize.namespace_user == user).execute()
```

Comment:
Can this be moved in in `_delete_user_linked_data`. 
Do you need a transaction in this case if it's only one row?

Reply:
Nope, I moved it into `_delete_user_linked_data`